<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boundaryData</class>
 <widget class="QFrame" name="boundaryData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="Lbl_H1">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;East Face&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_PatchLoc">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Patch location</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Cmb_PatchLoc">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Use this to define the type of patch</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <item>
      <property name="text">
       <string>Standard</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Cmb_PatchStdLoc">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Choose the standard patch for this boundary</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <item>
      <property name="text">
       <string>Entry</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exit</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Top</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bottom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_VelBC">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Velocity</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Cmb_UBC">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Choose type of velocity boundary condition</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <item>
      <property name="text">
       <string>Select velocity boundary condition</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>General: SW solutions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inlet: Constant velocity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inlet: Moving wall (OSU Flume)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inlet: Moving wall (General flume)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Outlet: zeroGradient</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Outlet: inletOutlet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wall: noSlip</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="Led_WaterDisp"/>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_UploadFile">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Upload files related to the boundary patch</string>
         </property>
         <property name="toolTipDuration">
          <number>10000</number>
         </property>
         <property name="text">
          <string>Wall displacement</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="Led_WaterHeight"/>
       </item>
       <item>
        <widget class="QPushButton" name="Btn_UploadFile_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Upload files related to the boundary patch</string>
         </property>
         <property name="toolTipDuration">
          <number>10000</number>
         </property>
         <property name="text">
          <string>Water height</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="Cmb_WaveType">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Types of wave inputs</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <item>
      <property name="text">
       <string>Airy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stokes2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stokes5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solitary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="GLY_Vel">
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="DSpBx_UMeanY">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Mean velocity (y-direction)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="DSpBx_DirnZ">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Wave direction vector (z-component)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="DSpBx_OZ">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Wave origian (z-coordinate)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="DSpBx_OY">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Wave origian (y-coordinate)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Lbl_Origin">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Origin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="DSpBx_DirnY">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Wave direction vector (y-component)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Lbl_Umean">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UMean</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="DSpBx_UMeanX">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Mean velocity (x-direction)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="DSpBx_UMeanZ">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Mean velocity (z-direction)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Lbl_Dirn">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Direction</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="DSpBx_DirnX">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Wave direction vector (x-component)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="DSpBx_OX">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Wave origian (x-coordinate)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Lbl_WavePara">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Length, Amplitude, 
Phase, Angle</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="Led_WavePara">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Wave input parameters</string>
       </property>
       <property name="toolTipDuration">
        <number>10000</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>0, 0, 0, 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Lbl_PresBC">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pressure</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Cmb_PresBC">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Choose type of pressure boundary condition</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
     <item>
      <property name="text">
       <string>Default choice</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>zeroGradient</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fixedValue</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLY_Pres" stretch="1,0">
     <item>
      <widget class="QLabel" name="Lbl_Pres">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="DSpBx_Pres">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Value of pressure</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="VSp">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>73</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
